src/01-functions.js

```diff
@@ -39,7 +43,9 @@ function greet() {}
  *  add(10, 20);
  *  //> 30
  */
-function add() {}
+function add(a,b) {
+  return a+b
```

Comment:
This code works great but when working with arithmetic operators lets space them out.  Same with out function parameters `(a,b)` => ` (a, b)` and  ` a+b ` => ` a + b `